### PR TITLE
Imported yaml-cpp 0.7.0

### PR DESCRIPTION
**Pull request by forthommel**
This PR imports version 0.7.0 of the commonly used `yaml-cpp` into the CMS environment.

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: yaml-cpp.spec [hunk 1]
```diff
@@ -0,0 +1,26 @@
+### RPM external yaml-cpp 0.7.0
+
+Source: https://github.com/jbeder/yaml-cpp/archive/refs/tags/%{n}-%{realversion}.tar.gz
+
+BuildRequires: cmake ninja
+
+%prep
+%setup -n %{n}-%{realversion}
```
Comment (smuzaffar) [inline yaml-cpp.spec]: @forthommel , please use `%{n}-%{n}-%{realversion}`

File: yaml-cpp.spec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+%build
+rm -rf ../build
+mkdir ../build
+cd ../build
+
+cmake ../%{n}-%{realversion} \
```
Comment (smuzaffar) [inline yaml-cpp.spec]: same here `%{n}-%{n}-%{realversion}`

File: yaml-cpp.spec (hunk 1 above)
Comment (forthommel) [reply yaml-cpp.spec]: Indeed, sorry for the false start!

Comment (forthommel) [reply yaml-cpp.spec]: @smuzaffar I also updated the `cmake` source path. I guess this is also needed, right?

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test for el8_ppc64le_gcc11

**Comment by smuzaffar** [comment]
please test for el8_aarch64_gcc11

**Comment by smuzaffar** [comment]
please test for el9_amd64_gcc11

**Comment by smuzaffar** [comment]
please test for slc7_amd64_gcc11

**Review by smuzaffar — APPROVED** [review]

**Comment by smuzaffar** [comment]
+externals
looks good

**Comment by forthommel** [comment]
Many thanks for the extremely fast interaction (and follow-up PR), @smuzaffar !

**Pull request merged**